docs/Tutorials/Misc/DesktopApp.md

```diff
@@ -49,3 +49,19 @@ The following is a basic example of a `.bat` file which could be double-clicked
 powershell.exe -noprofile -windowstyle hidden -command .\you-server-script.ps1
 exit
 ```
+
+## Using Chromium instead of Internet Explorer
+
+The default WPF Browser Element used by Pode is based on the system internal Internet Explorer API which is also bound to the same Javascript and Webbrowser Limitations as Internet Explorer itself.
```

Comment:
Don't want to be "that guy", but is there any chance a comma could be place about here: `Internet Explorer API, which is also bound`? Sorry!

Reply:
Thanks, as non native speaker I tend to focus more on the content than on the gramma. Fixed.